src/utils/calculateTime.ts

```diff
@@ -15,3 +24,16 @@ export const getPreviousWeek = (): number => {
 
   return secondsSinceEpoch
 }
+
+export const getPreviousMonth = (): number => {
+  const dayInSecs = 24 * 60 * 60
+  const now = new Date()
+  const previousMonth =
+    Math.floor(
+      (now.getTime() - dayInSecs * 30 * 1000) / 1000 / dayInSecs
+    ) *
+      dayInSecs +
+    dayInSecs * 3
```

Comment:
this is weird, I don't know about the calculation, but by reading this seems make no sense it's a previous month cc @xanderjl @sandbochs